### DISCUSSION TITLE
LookUp - Exibir informações

### DISCUSSION THREAD

**Discussion by gtrabach** [po-ui/po-angular#1646] [Tire suas dúvidas]
Estou criando um campo do tipo LookUp através da propriedade _searchService_ do componente **Dynamic Form**.

O campo funciona normalmente ao preencher o mesmo manualmente, mas ao informar o valor para o Form via carga (exemplo: visualização ou alteração de uma página), a informação do campo não é carregada. Estranhamente se eu clico na lupa ela já está com o registro certo filtrado, mas o mesmo não está selecionado.

Quando o formulário é carregado, o campo é exibido em branco:

![image](https://user-images.githubusercontent.com/46979335/221596020-5a20c1fc-f844-4f2c-9d16-4819bc4fb020.png)

Ao clicar na lupa, exibe o conteúdo filtrado que deveria estar sendo exibido no campo:

![image](https://user-images.githubusercontent.com/46979335/221596080-24d99700-976e-440f-a5a0-0c6686b9fff1.png)

Sou iniciante em POUI, então não sei se descrevi bem o meu problema rs.

Exemplo de como está a minha propriedade p-fields:

```
this.fieldsPreCadastro.push({
	property: 'ZVB_FILIAL', 
	label: 'Filial do Sistema', 
	gridColumns: 4, 
	order: 1, 
	searchService: "https://urldaminhaapi/consulta/sm0"
	fieldValue: 'codigo',
	columns: [{property: 'codigo', label: 'Codigo'}, {property: 'descricao', label: 'Descricao'}],
	format: ["codigo", "descricao"],
	params: {chave: '0101'},
	readonly: true
})
```

Obrigado pela ajuda desde já.

**Comment by alinelariguet** [comment]
Boa tarde @gtrabach, você poderia simular no stackblitz a situação reportada para que possamos analisar?
segue link: https://stackblitz.com/edit/po-ui
Esse discussion pode ser convertido em issue.
Att.

**Comment by gtrabach** [comment]
Boa tarde, Aline!

Consegui resolver, era um problema no comportamento da API. Efetuei o ajuste na API e removi a propriedade params e o componente funcionou corretamente.

Obrigado.
